salesagility/SuiteCRM
# Fixed #7333 - Unminify JavaScript in the SuiteCRM repository

## Description



Ideally, we shouldn't be including any autogenerated files with git. This PR removes the minified JS files that are built when running JS repair.

TODO: 
- [X] Create robo task for minifying JS.
- [x] Fix Travis.

## Motivation and Context

Issue reference: #7333

## How To Test This

1. Run ```./vendor/bin/robo build:js```
2. Install CRM and check that the removed files have been rebuilt.

## Types of changes


### Final checklist
